devicon.json

```
@@ -7026,6 +7026,27 @@
             }
         ]
     },
+    {
+        "name": "stata",
+        "altnames": [],
+        "tags": [
+            "analysis",
+            "data",
+            "data-science",
+            "software",
+            "statistics"
+        ],
+        "versions": {
+            "svg": [
+                "original-wordmark"
+            ],
+            "font": [
+                "original-wordmark"
+            ]
+        },
+        "color": "#195f92",
+        "aliases": []
```

Should we add aliases?
```suggestion
        "aliases": [
            {
                "base": "original-wordmark",
                "alias": "plain-wordmark"
            }
        ]
```